quinn-proto/src/connection/mod.rs

```diff
@@ -3309,6 +3385,26 @@ impl Connection {
         self.path.mtud.on_peer_max_udp_payload_size_received(
             u16::try_from(self.peer_params.max_udp_payload_size.into_inner()).unwrap_or(u16::MAX),
         );
+
+        {
+            self.peer_ack_timestamp_cfg = if let (Some(max_timestamps_per_ack), Some(exponent)) = (
+                params.max_recv_timestamps_per_ack,
+                params.receive_timestamps_exponent,
+            ) {
+                for space in self.spaces.iter_mut() {
+                    space
+                        .pending_acks
+                        .set_receiver_timestamp(max_timestamps_per_ack.0 as usize);
+                }
+                Some(AckTimestampsConfig {
+                    exponent,
+                    max_timestamps_per_ack,
+                    basis: self.epoch,
+                })
+            } else {
+                None
+            };
+        }
```

Comment:
Suggest spelling this as:

`let (Some(max_timestamps_per_ack), Some(exponent)) = (params.max_recv_timestamps_per_ack, params.receive_timestamps_exponent) else { return None };`